We have no pytest-sugar packaged yet.
It is not needed to run tests.

--- typer-0.12.4/requirements-tests.txt.orig
+++ typer-0.12.4/requirements-tests.txt
@@ -4,7 +4,6 @@
 pytest-cov >=2.10.0,<5.0.0
 coverage[toml] >=6.2,<8.0
 pytest-xdist >=1.32.0,<4.0.0
-pytest-sugar >=0.9.4,<0.10.0
 mypy ==1.4.1
 # Needed explicitly by typer-slim
 rich >=10.11.0
